### PR TITLE
Implementation of cummax on backend

### PR DESCRIPTION
**Pull request by tvsharish**
I have implemented cummax on torch, numpy, paddle, Tensorflow and Jax backend and also added function in `test_statistical.py` which will close issue #14098.  As suggested by @ZiadAmerr , I have pulled new PR from new branch so that new test fixtures can run.

### CODE REVIEW

File: ivy/functional/backends/paddle/statistical.py [hunk 1]
```diff
@@ -1,7 +1,10 @@
 # global
 
 torch_scatter = None
-from typing import Union, Optional, Sequence
+
+from typing import Union, Optional, Sequence, Tuple
+
+import paddle
```
Comment (ZiadAmerr) [inline ivy/functional/backends/paddle/statistical.py:7]:
```suggestion
```

This line was causing a linting error

Comment (tvsharish) [reply ivy/functional/backends/paddle/statistical.py:7]: Okay. I'll make the change.

**Comment by ZiadAmerr** [comment]
Looks like the tests are failing anyways, so we will merge the other PR. I am going to close this now, please fix the linting error and add any extra additions to the other PR and we are ready to merge :)

**Comment by ZiadAmerr** [comment]
Sorry for this inconvenience though.

**Pull request closed without merging**